keyboards/longnald/corin/info.json

```diff
@@ -0,0 +1,77 @@
+{
+  "keyboard_name": "Corin",
+  "url": "https://github.com/longnald/corin",
+  "maintainer": "Seaton Jiang <seaton@vtrois.com>",
+  "width": 28.54,
+  "height": 9.58,
```

Comment:
```suggestion
  "width": 15,
  "height": 5,
```

Reply:
In reference to [info.json Format](https://docs.qmk.fm/#/reference_info_json?id=infojson-format), don't the width and height refer to the size of the pcb?

Reply:
Yes, but as noted in that doc, it's measured in Key Units. An alpha key is `1`, ANSI Enter is `2.25`, etc. A standard 60% keyboard (like a Poker or DZ60) is `"width": 15, "height": 5`.